shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/ApiArrivalsWithSplits.scala

```diff
@@ -22,41 +22,20 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
     with Updatable[ApiFlightWithSplits]
     with WithLastUpdated {
 
-  def totalPaxFromApi: Option[TotalPaxSource] = splits.collectFirst {
+  def bestPaxFromApi: Option[PaxSource] = splits.collectFirst {
     case splits if splits.source == ApiSplitsWithHistoricalEGateAndFTPercentages =>
-      TotalPaxSource(Option(Math.round(splits.totalPax).toInt), ApiFeedSource)
+      PaxSource(ApiFeedSource, Passengers(Option(splits.totalPax), Option(splits.transPax)))
   }
 
-  def totalPaxFromApiExcludingTransfer: Option[TotalPaxSource] =
-    splits.collectFirst { case splits if splits.source == ApiSplitsWithHistoricalEGateAndFTPercentages =>
-      TotalPaxSource(Option(Math.round(splits.totalExcludingTransferPax).toInt), ApiFeedSource)
-    }
-
-  def pcpPaxEstimate: TotalPaxSource =
-    totalPaxFromApiExcludingTransfer match {
+  def bestPaxSource: PaxSource =
```

Comment:
Can we check if we always add api pax nos to an `Arrival` or if they're only added when they're considered valid? 
If they're always added then this function has a potential bug: when api is invalid it calls `bestPaxEstimate` which could still return the api number if we decided to change the logic in that function
If they're only added when valid then can just directly call `apiFlight.bestPaxEstimate`

Reply:
Did you manage to find the answer to this question above?^